.github/workflows/integration.yml

```diff
@@ -35,7 +35,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php-versions: ['7.3', '7.4']
+        php-versions: ['7.4', '8.0', '8.1']
```

Comment:
Do we want to drop PHP 7.3 support? I'd be ok with doing so and we could modernize a few things, like type declarations for class properties and use fancy new stuff like arrow functions and unpacking inside arrays.

On the other hand, this means that Userli no longer runs on Debian Buster (oldstable) systems, which is ok from my point of view.

Reply:
PHP 7.3 is EOL since 2022. For me it is okay to drop.